client/state/ui/reader/sidebar/reducer.js

```diff
@@ -27,17 +28,36 @@ export function isTagsOpen( state = false, action ) {
 	return state;
 }
 
-export function isNetworkOpen( state = false, action ) {
+export function isFollowingOpen( state = false, action ) {
 	switch ( action.type ) {
-		case READER_SIDEBAR_NETWORK_TOGGLE:
+		case READER_SIDEBAR_FOLLOWING_TOGGLE:
 			return ! state;
 	}
 
 	return state;
 }
 
+export function openOrganizations( state = [], action ) {
+	switch ( action.type ) {
+		case READER_SIDEBAR_ORGANIZATIONS_TOGGLE: {
+			const opened = state;
+			const index = opened.indexOf( action.organizationId );
```

Comment:
This block looks essentially like the lodash `xor` function:
https://lodash.com/docs/4.17.15#xor
Readability, though, might be nicer here with the native functions.  Up to you.

Reply:
`xor` is definitely more straight forward.